sdk/textanalytics/azure-ai-textanalytics/samples/authoring/dummy_project.py

```diff
@@ -0,0 +1,83 @@
+dummy_project = {
```

Comment:
Let's rename this file to sample_project if it's to be used with the samples. Also, should this be a JSON file instead of a .py?